benchmarks/prompt_eval/.env.example

```
@@ -0,0 +1,4 @@
+
+BASE_URLs=https://example/v1/;https://example2/v1/;
```

dotenv-linter (4.0.0)

[warning] [LowercaseKey] The BASE_URLs key should be in uppercase

(LowercaseKey)